buildscripts/semantic-convention/templates/registry/otel4s/attributes/experimental/SemanticAttributes.scala.j2

```diff
@@ -0,0 +1,73 @@
+{%- macro to_scala_key_type(attribute) -%}
+    {{ attribute.type | instantiated_type | map_text("scala_key_type") }}
+{%- endmacro %}
+
+{%- macro stable_ref(attribute) -%}
+    {{ params.packages.stable }}.{{ ctx.root_namespace | pascal_case }}Attributes.{{ attribute.name | pascal_case }}
+{%- endmacro -%}
+
+{%- set object_name = ctx.root_namespace | pascal_case ~ "ExperimentalAttributes" -%}
+
+/*
+ * Copyright 2023 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package {{ params.packages.experimental }}
+
+import org.typelevel.otel4s.AttributeKey
```

Comment:
just an idle thought (and probably not worth it): you could theoretically split the package declaration in two and remove the import

Reply:
Sounds good. I will explore this option in the follow-up PR.